lib/__tests__/extends.test.mjs

```diff
@@ -68,7 +67,7 @@ it('extending a config that is overridden', async () => {
 });
 
 describe('extending a config from process.cwd', () => {
-	safeChdir(__dirname);
+	safeChdir(new URL('.', import.meta.url));
```

Comment:
I noticed that in other lines `fileURLToPath` is used. Why it's omitted here?

Reply:
Now `safeChdir()` also allows an `URL` argument like many Node.js APIs do, e.g. [`fs.readFile()`](https://nodejs.org/api/fs.html#fsreadfilepath-options-callback).